spec/models/post_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+describe Post, type: :model do
+  subject do
+    first_user = User.create(name: 'Atif', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                             bio: 'A pakistani micronaut.', posts_counter: 0)
+    Post.create(author: first_user, title: 'Hello', text: 'This is a personal post', likes_counter: 0,
+                comments_counter: 0)
+  end
```

Comment:
- [ ] As requested by the previous reviewer, As required for the project to test any custom methods in the models, kindly ensure that you test the `#five_recent_comments` and `#update_posts_counter` methods in the Post model :pray:

Reply:
Thank you for pointing out the missing test. I will definitely add it